src/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools/Internal/PrecompileDispatchCommand.cs

```diff
@@ -104,18 +80,13 @@ private int Execute()
                 dispatchArgs.Add(Options.ConfigureCompilationType.Value());
             }
 
-            if (Options.GeneratePdbOption.HasValue())
-            {
-                dispatchArgs.Add("--generate-pdbs");
-            }
-
             var toolName = typeof(Design.Program).GetTypeInfo().Assembly.GetName().Name;
             var dispatchCommand = DotnetToolDispatcher.CreateDispatchCommand(
                 dispatchArgs,
                 TargetFramework,
                 Configuration,
-                outputPath: OutputPath,
-                buildBasePath: BuildBasePathOption.Value(),
+                outputPath: outputPaths.RuntimeOutputPath,
```

Comment:
This works pretty well in the publish + post build scenario. Publish builds first to the default locations for a given configuration \ framework option. Prebuild is too early since no binaries exist at this stage and we'd be responsible for the build. This gives us a way out which works well for our primary scenario.